### PR TITLE
clustering yaml creayed 

### PR DESCRIPTION
**Pull request by giuliaelgarcia**
Created clustering yaml file. 
Please check line 267 for default and check formating from line 140 onwards (when there are 2 default values)

### CODE REVIEW

File: docs/yaml_docs/pipeline_clustering_yml.md [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+When pseudo_seurat is set to true then a python implementation of Suerat runs (Seurat::FindMarkers written by CRG)
+
+  - <span class="parameter">markerspecs:</span> <br>
+     - <span class="parameter">rna:</span><br>
+       - <span class="parameter">run </span> `Boolean`, Default: True<br>
+       - <span class="parameter">layer </span> `String`, Default: logged_counts<br>
+       - <span class="parameter">method </span> `String`, Default: t-test_overestim_var<br>
+       Options include: ‘logreg’, ‘t-test’, ‘wilcoxon’, ‘t-test_overestim_var’
+       - <span class="parameter">mincels </span> `Integer`, Default: t-10<br>
+       If the number of clusters contains less than the number of cells maker analysis is not necessary.
```
Comment (bio-la) [inline docs/yaml_docs/pipeline_clustering_yml.md:204]: i don't understand this, please check the documentation of scanpy.rank_genes to explain this parameter

File: docs/yaml_docs/pipeline_clustering_yml.md [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+       - <span class="parameter">pseudo_seurat </span> `Boolean`, Default: False<br>
+       - <span class="parameter">minpct </span> `Float`, Default: 0.1<br>
+       This parameter only matters if pseudo_seurat is set to True 
+       - <span class="parameter">threshuse </span> `Float`, Default: 0.25<br>
+       This parameter only matters if pseudo_seurat is set to True
```
Comment (bio-la) [inline docs/yaml_docs/pipeline_clustering_yml.md:209]: check my previous corrections "this parameter is mandatory if..."

**Review by bio-la — CHANGES_REQUESTED** [review]
please pull again the branch (i've added some fixes) and check my comments.
I've added the file to the index so you can see the documentation rendered and check how the structure looks.
please be careful with spelling typos

File: docs/yaml_docs/pipeline_clustering_yml.md [hunk 3]
```diff
@@ -228,7 +228,7 @@ When pseudo_seurat is set to True then a [python implementation](https://github.
     - <span class="parameter">layer </span> `String`, Default: logged_counts<br>
       Which layer stores counts for differential expression test. 
        Options include logged_counts, signac_norm , and logTF_norm,logIDF_norm
-    - <span class="parameter">mincels </span> `Integer`, Default: t-10<br>
+    - <span class="parameter">mincels </span> `Integer`, Default: 10<br>
        If the number of clusters contains less than the number of cells maker analysis is not necessary.
```
Comment (bio-la) [inline docs/yaml_docs/pipeline_clustering_yml.md:232]: Min number of cells in a cluster. If the cluster contains less than this number of cells, the marker analysis won't be run.

**Review by bio-la — COMMENTED** [review]
please spell check!

File: docs/yaml_docs/pipeline_clustering_yml.md [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+
+  - <span class="parameter">markerspecs:</span> <br>
+     - <span class="parameter">rna:</span><br>
+       - <span class="parameter">run </span> `Boolean`, Default: True<br>
+       - <span class="parameter">layer </span> `String`, Default: logged_counts<br>
+         Which layer stores counts for differential expression test.
+       - <span class="parameter">method </span> `String`, Default: t-test_overestim_var<br>
+       Options include: ‘logreg’, ‘t-test’, ‘wilcoxon’, ‘t-test_overestim_var’
+       - <span class="parameter">mincels </span> `Integer`, Default: 10<br>
+       Minimal number of cells in a cluster. If the cluster contains less than this number of cells, the marker analysis won't be run.
```
Comment (deevdevil88) [inline docs/yaml_docs/pipeline_clustering_yml.md:207]: I would say " Marker analysis  is run for clusters  >= mincells. If a cluster ncells < mincells , then the cluster is excluded from marker analysis

File: docs/yaml_docs/pipeline_clustering_yml.md [hunk 5]
```diff
@@ -0,0 +1,276 @@
...
+When pseudo_seurat is set to True then a [python implementation](https://github.com/DendrouLab/panpipes/blob/main/panpipes/python_scripts/run_find_markers_multi.py) of `Seurat:::FindMarkers` is run
+
+  - <span class="parameter">markerspecs:</span> <br>
+     - <span class="parameter">rna:</span><br>
+       - <span class="parameter">run </span> `Boolean`, Default: True<br>
+       - <span class="parameter">layer </span> `String`, Default: logged_counts<br>
+         Which layer stores counts for differential expression test.
+       - <span class="parameter">method </span> `String`, Default: t-test_overestim_var<br>
+       Options include: ‘logreg’, ‘t-test’, ‘wilcoxon’, ‘t-test_overestim_var’
+       - <span class="parameter">mincels </span> `Integer`, Default: 10<br>
```
Comment (deevdevil88) [inline docs/yaml_docs/pipeline_clustering_yml.md:206]: type change to mincells

File: docs/yaml_docs/pipeline_clustering_yml.md [hunk 6]
```diff
@@ -0,0 +1,276 @@
...
+       - <span class="parameter">pseudo_seurat </span> `Boolean`, Default: False<br>
+       - <span class="parameter">minpct </span> `Float`, Default: 0.1<br>
+       This parameter is mandatory if pseudo_seurat is set to True 
+       - <span class="parameter">threshuse </span> `Float`, Default: 0.25<br>
+       This parameter is mandatory if pseudo_seurat is set to True 
+ - <span class="parameter">prot:</span><br>
+   - <span class="parameter">run </span> `Boolean`, Default: True<br>
+   - <span class="parameter">layer </span> `String`, Default: clr<br>
+       Which layer stores counts for differential expression test.
+   - <span class="parameter">mincels </span> `Integer`, Default: 10<br>
```
Comment (deevdevil88) [inline docs/yaml_docs/pipeline_clustering_yml.md:217]: typo it should be mincells

File: docs/yaml_docs/pipeline_clustering_yml.md [hunk 7]
```diff
@@ -0,0 +1,276 @@
...
+       This parameter is mandatory if pseudo_seurat is set to True 
+   - <span class="parameter">threshuse </span> `Float`, Default: 0.25<br>
+       This parameter is mandatory if pseudo_seurat is set to True 
+
+ - <span class="parameter">atac:</span><br>
+    - <span class="parameter">run </span> `Boolean`, Default: False<br>
+    - <span class="parameter">layer </span> `String`, Default: logged_counts<br>
+      Which layer stores counts for differential expression test. 
+       Options include logged_counts, signac_norm , and logTF_norm,logIDF_norm
+    - <span class="parameter">mincels </span> `Integer`, Default: 10<br>
```
Comment (deevdevil88) [inline docs/yaml_docs/pipeline_clustering_yml.md:231]: typo and change expalnaation as above

File: docs/yaml_docs/pipeline_clustering_yml.md [hunk 8]
```diff
@@ -0,0 +1,276 @@
...
+        Options include: ‘logreg’, ‘t-test’, ‘wilcoxon’, ‘t-test_overestim_var’
+    - <span class="parameter">pseudo_seurat </span> `Boolean`, Default: False<br>
+    - <span class="parameter">minpct </span> `Float`, Default: 0.1<br>
+       This parameter is mandatory if pseudo_seurat is set to True 
+    - <span class="parameter">threshuse </span> `Float`, Default: 0.25<br>
+       This parameter is mandatory if pseudo_seurat is set to True 
+
+
+ - <span class="parameter">multimodal:</span><br>
+   - <span class="parameter">mincels </span> `Integer`, Default:10<br>
```
Comment (deevdevil88) [inline docs/yaml_docs/pipeline_clustering_yml.md:243]: typo and explanantion change

File: docs/yaml_docs/pipeline_clustering_yml.md [hunk 9]
```diff
@@ -0,0 +1,276 @@
...
+       This parameter is mandatory if pseudo_seurat is set to True
+
+
+ - <span class="parameter">spatial:</span><br>
+   - <span class="parameter">run </span> `Boolean`, Default: True<br>
+   - <span class="parameter">layer </span> `String`, Default: norm_pearson_resid<br>
+       Options include logged_counts, signac_norm , and logTF_norm,logIDF_norm
+   - <span class="parameter">method </span> `String`, Default: t-test_overestim_var<br>
+        Options include: ‘logreg’, ‘t-test’, ‘wilcoxon’, ‘t-test_overestim_var’
+   - <span class="parameter">mincels </span> `Integer`, Default: 10<br>
```
Comment (deevdevil88) [inline docs/yaml_docs/pipeline_clustering_yml.md:260]: typo and expalantion change

**Review by deevdevil88 — CHANGES_REQUESTED** [review]
some typos and  mincells explanation needs to be changed

**Review by bio-la — COMMENTED** [review]
- clarified modalities
- removed wrong parameters

**Review by bio-la — APPROVED** [review]

**Review by deevdevil88 — APPROVED** [review]

**Pull request merged**